src/liquid/place/move_form.html

```diff
@@ -54,7 +54,12 @@ <h1 class="subtitle">To</h1>
 <script type="text/javascript">
   function handleResponse(event) {
     const response = event?.detail?.xhr?.responseText;
-    if (response && response.includes('data-success="true"')) {
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(response, 'text/html');
+
+    // Find the element that indicates success
+    const successElement = doc.querySelector('[data-success="true"]');
+    if (successElement) {
```

Comment:
Should something happen when !successElement?

Reply:
Yeah, when no success (no data-success="true"), there is already the error block displaying an error notification...

```
  {% if error %}
    <div class="notification is-danger" style="margin: 0;">
      {{ error }}
    </div>
  {% endif %}

  {% if success %}
    <div data-success="true"></div>
  {% endif %}
  ```